renovate.json

```diff
@@ -6,5 +6,19 @@
     "pom.xml",
     "google-cloud-pom-parent/pom.xml",
     "google-cloud-jar-parent/pom.xml"
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["^.kokoro/nightly/graalvm-native.cfg$",
+        "^.kokoro/presubmit/graalvm-native-.*.cfg$",
```

Comment:
Done. Verified with this test setup that the regex without the escape works as expected: https://github.com/mpeddada1/renovate-bot-test/pull/7/files.

Both formats seem to be accepted since we have an example that doesn't include `\\`: https://github.com/googleapis/google-api-java-client-services/blob/d0b6a8fca7ab5b60c79926ad7919bcae223a099f/renovate.json#L33